libs/feature/editor/src/lib/expressions.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { evaluate, ExpressionEvaluator } from './expressions'
+import { EditorFieldConfig } from './models/fields.model'
+
+const SAMPLE_CONFIG: EditorFieldConfig = {
+  formFieldConfig: {
+    labelKey: 'Metadata title',
+    type: 'text',
+  },
+  model: 'myModel',
+}
+
+describe('expressions', () => {
+  let evaluator: ExpressionEvaluator
+  describe('operators', () => {
+    describe('dateNow', () => {
+      beforeEach(() => {
+        evaluator = evaluate('${dateNow()}')
+      })
+      it('returns the current time at evaluation', () => {
+        expect(evaluator({ config: SAMPLE_CONFIG, value: 'bla' })).toEqual(
+          new Date()
```

Comment:
You don't need to mock the Date API for the test to "always" work?

Edit: fixed in last commit